plugins/scripts/inject-config.js

```diff
@@ -2,6 +2,27 @@
 const fs = require('fs');
 const path = require('path');
 
+function createJsonWithDefaultValues(optimoveCredentials, optimoveMobileCredentials , inAppConsentStrategy) {
+    if (!isString(inAppConsentStrategy) || isEmpty(inAppConsentStrategy)) { 
```

Comment:
is this check necessary? when we validate config we already make sure in-app strategy is good

Reply:
yes I agree, I thought to double check here, do you think we should get rid of it?